migrate.go

```diff
@@ -15,6 +15,8 @@ import (
 )
 
 var (
+	// ErrNoMigrationsFound when no migrations have been found.
+	ErrNoMigrationsFound = errors.New("no migrations found")
```

Comment:
```suggestion
	ErrNoMigrationFiles = errors.New("no migrations files found")
```

Wdyt about renaming this? The rationale is there may be "db migrations" and "file migrations", and being explicit avoids the confusion that no migration _file_ was found.

Reply:
👋 @mfridman. No problem at all. Thanks for the review!